src/Symfony/Component/DependencyInjection/Tests/Dumper/PhpDumperTest.php

```diff
@@ -236,11 +236,11 @@ public function testDumpAsFiles()
             ->addError('No-no-no-no');
         $container->compile();
         $dumper = new PhpDumper($container);
-        $dump = print_r($dumper->dump(['as_files' => true, 'file' => __DIR__, 'hot_path_tag' => 'hot', 'inline_factories_parameter' => false, 'inline_class_loader_parameter' => false]), true);
+        $dump = print_r($dumper->dump(['as_files' => true, 'file' => __DIR__, 'hot_path_tag' => 'hot', 'inline_factories_parameter' => false, 'inline_class_loader_parameter' => false, 'build_time' => 1588866825]), true);
         if ('\\' === \DIRECTORY_SEPARATOR) {
             $dump = str_replace('\\\\Fixtures\\\\includes\\\\foo.php', '/Fixtures/includes/foo.php', $dump);
         }
-        $this->assertStringMatchesFormatFile(self::$fixturesPath.'/php/services9_as_files.txt', $dump);
+        $this->assertStringEqualsFile(self::$fixturesPath.'/php/services9_as_files.txt', $dump);
```

Comment:
We can't use a regex anymore because the file to compare to have become too big (`preg_match(): Compilation failed: regular expression is too large at offset 35752`). I guess this strategy is useful when merging from lower branches? Do someone have a better idea than strict comparison or creating a new file? 😕

Reply:
Tests are failing: I need to recheck the strategy.